src/main/java/seedu/address/ui/DeliverableDetailsPanel.java

```diff
@@ -42,7 +42,10 @@ public DeliverableDetailsPanel(Deliverable deliverable) {
         deliverable.getContacts().value.ifPresentOrElse(contacts -> {
             for (String contact : contacts.split(",")) {
                 String cleanedContact = contact.trim().replaceAll("\\s+", " ");
-                contactList.getChildren().add(new Label("- " + cleanedContact));
+                Label label = new Label("-" + cleanedContact);
```

Comment:
Do we still want the space i.e. "- "?

Reply:
Its cos i decided to wrap contacts also but with the space there the entire thing will go to next line. Do you think its ok?